ui/dev.go

```diff
@@ -6,7 +6,7 @@ package ui
 import "embed"
 
 var (
-	//go:embed index.html
+	//go:embed dev.html
```

Comment:
TODO: this doesnt work yet. will  have to redo the development workflow in another PR